Index: cbeditor.cpp
===================================================================
--- cbeditor.cpp	(revision 9861)
+++ cbeditor.cpp	(working copy)
@@ -1001,6 +1001,9 @@
     m_pData->m_encoding = wxFontMapper::GetEncodingFromName(
         Manager::Get()->GetConfigManager(_T("editor"))->Read(_T("/default_encoding"), wxEmptyString) );
 
+    for (int marker = 0 ; marker <= wxSCI_MARKNUM_LASTUNUSED ; ++marker)
+        control->MarkerDefine(marker, wxSCI_MARK_EMPTY);
+
     return control;
 }
 
Index: wxscintilla/include/wx/wxscintilla.h
===================================================================
--- wxscintilla/include/wx/wxscintilla.h	(revision 9861)
+++ wxscintilla/include/wx/wxscintilla.h	(working copy)
@@ -123,6 +123,9 @@
 #define wxSCI_MARK_CHARACTER 10000
 
 // Markers used for outlining and changed column.
+/* C::B begin */
+#define wxSCI_MARKNUM_LASTUNUSED 22
+/* C::B end */
 #define wxSCI_MARKNUM_CHANGEUNSAVED 23
 #define wxSCI_MARKNUM_CHANGESAVED 24
 #define wxSCI_MARKNUM_FOLDEREND 25
