Index: cbeditor.cpp
===================================================================
--- cbeditor.cpp	(revision 9861)
+++ cbeditor.cpp	(working copy)
@@ -1001,6 +1001,9 @@
     m_pData->m_encoding = wxFontMapper::GetEncodingFromName(
         Manager::Get()->GetConfigManager(_T("editor"))->Read(_T("/default_encoding"), wxEmptyString) );
 
+    for (int marker = 0 ; marker <= wxSCI_MARKNUM_LASTUNUSED ; ++marker)
+        control->MarkerDefine(marker, wxSCI_MARK_EMPTY);
+
     return control;
 }
 
